.gitignore

```diff
@@ -32,3 +32,6 @@
 /config/bbgo.yaml
 
 /pkg/server/assets.go
+
+node_modules
+yarn.lock
```

Comment:
we should include the lock?

Reply:
yep, it seems you are right. 
https://classic.yarnpkg.com/en/docs/yarn-lock/